### PR TITLE
fix(issues)：fix issues of bug tag

### PR DESCRIPTION
**Pull request by wzh425**

### CODE REVIEW

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Mc.Service.Admin.Application.MessageRecords.EventHandler;
+
+public class RetryEmailMessageEventHandler
+{
+    private readonly IEmailAsyncLocal _emailAsyncLocal;
```
[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: I need to set the parameters of the channel before sending

File: src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetrySmsMessageEventHandler.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Mc.Service.Admin.Application.MessageRecords.EventHandler;
+
+public class RetrySmsMessageEventHandler
+{
+    private readonly IAliyunSmsAsyncLocal _aliyunSmsAsyncLocal;
```
[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetrySmsMessageEventHandler.cs:8]: I need to set the parameters of the channel before sending

File: src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs [hunk 3]
```diff
@@ -50,5 +50,7 @@ public async Task HandleEventAsync(UpdateMessageTaskStatusEvent eto)
             messageTask.SetResult(MessageTaskStatuses.Fail);
         }
         await _repository.UpdateAsync(messageTask);
+        await _repository.UnitOfWork.SaveChangesAsync();
```
[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs:53]: Currently, updating data in the EventHandler of domain events added by the aggregation root requires manual UOW

File: src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: EmailAsyncLocal = 邮箱异步本地？

File: src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetrySmsMessageEventHandler.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetrySmsMessageEventHandler.cs:8]: 异步本地？

File: src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs:53]: @zhenlei520 PTAL

File: src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: AsyncLocalOptionsResolveContributor  异步本地Options解析器

[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: EmailAsyncLocalOptions

[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: 异步本地 是什么意思

[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: 异步本地存储，跨线程保留数据

[the remark this answers is not in the document]
Comment (doddgu) [reply src/Services/Masa.Mc.Service/Application/MessageRecords/EventHandler/RetryEmailMessageEventHandler.cs:8]: 一个类名无法直接表达这个类要做的事情，那这个类名就是不对的，你再想一想

File: src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Services/Masa.Mc.Service/Domain/MessageTasks/EventHandler/UpdateMessageTaskStatusEventHandler.cs:53]: > @zhenlei520他问

it is recommended to remove it and wait for the 0.5.0-preview.5 version

**Pull request closed without merging**